Future-Hope/Future-Hope/OnBoardingViews/EmailSignUpViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EmailSignUpViewController.swift
+//  Future-Hope
+//
+//  Created by Hector Steven on 8/20/19.
+//  Copyright © 2019 Hector Steven. All rights reserved.
+//
+
+import UIKit
+
+import Firebase
+
+import MaterialComponents.MaterialTextFields
+import MaterialComponents.MaterialButtons
+
+class EmailSignUpViewController: UIViewController {
+
+	
+	@IBOutlet var fullNameTextField: MDCTextField!
+	@IBOutlet var emailTextFields: MDCTextField!
+	@IBOutlet var passwordTextField: MDCTextField!
+	@IBOutlet var confirmPasswordTextField: MDCTextField!
+	
+
+	override func viewDidLoad() {
+        super.viewDidLoad()
+		
+//		navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Test", style: .plain, target: self, action: #selector(Test))
+		
+    }
+	
+//	@objc func Test() {
```

Comment:
What was this testing? Is this just here for debugging? Is it still being used occasionally? If not then it can probably be tossed to clean up the code.

Reply:
Yes, it needs to be tosed. I used it to test the fireabse password login.

Reply:
Cool. I'll go ahead and approve once it's tossed then.